aQute.libg/src/aQute/libg/pack200/Unpack200.java

```diff
@@ -0,0 +1,147 @@
+package aQute.libg.pack200;
```

Comment:
I am not sure why this live in `aQute.libg`. We should probably just put it in `biz.aQute.repository` as a `aQute.p2.packed` package.